uni/lib/redux/reducers.dart

```diff
@@ -67,6 +67,8 @@ AppState appReducers(AppState state, dynamic action) {
     return setUserFaculties(state, action);
   } else if (action is SetRestaurantsAction) {
     return setRestaurantsAction(state, action);
+  } else if(action is SetCalendarAction){
+    return setCalendarAction(state, action);
   }
```

Comment:
Beyond this, do not forget to register the `SetCalendarStatusAction` and call the appropriate reducer (which you do not have yet)!